src/modules/interaction.js

```diff
@@ -0,0 +1,65 @@
+let taskHolder = JSON.parse(localStorage.getItem('taskList')) || [];
+
+window.addEventListener('load', () => {
+  const checkbox = document.querySelectorAll('.checkbox[data-index]');
+
+  // checkbox functionality
+  checkbox.forEach((check) => {
+    const { index } = check.dataset;
+    const spanCheck = check.nextElementSibling;
+    check.addEventListener('click', (event) => {
```

Comment:
- [ ] **checkbox** should have the `change` event listener. Kindly remove the `click` and use `change` 👍🏾